### ISSUE TITLE
Add 'Required WP' and 'Required PHP' version numbers to file header at '/popup-maker.php'

### ISSUE THREAD

**Issue by rgadon107** [PopupMaker/Popup-Maker#622]
@danieliser To improve code readability, I recommend that the following 2 lines be added to the docblock header at `/popup-maker.php`: 

Require WP: 3.6
Require PHP: 5.2.17

From `class Popup_Maker`, the minimum required versions of WP and PHP are set as static methods. But someone opening the plugin code may not know where to find that information. It just makes it easier to find the minimum plugin requirements by adding that info to the plugin bootstrap file header. 

BTW, this is no different that what you do currently on the PUM extension plugins. There's already a line in the docblock header of each extension bootstrap file for 'Requires Popup Maker: {minimum-version-required}'. 

While you're at it, if you're planning to increase the minimum required PHP version to run the plugin (consistent with WP in 2019), it would make sense to add a version check for the PHP minimum on plugin install, and return an error message to notify users to upgrade their PHP version if they're running below the minimum requirement.

**Comment by danieliser** [comment]
No need, this is already in the readme where WP expects it to be for wordpress.org repo page (already shows it now https://wordpress.org/plugins/popup-maker), and already added it to readme https://github.com/PopupMaker/Popup-Maker/blob/release/1.8.0/readme.txt#L7-L9

Am I missing something here, I don't see any mention in WordPress.org guidelines for adding those to the plugin header comments themselves.

PS the static variables aren't for people to read neccessarily, that is what the readme.txt would ultimately be for.

They are for automatic comparison and disabling of the plugins when versions don't match up to what they need to be.

PPS. WP core will ultimately be adding in PHP version checks on a per plugin/theme basis. IE they will build in checks based on the version listed in readme.txt and safely disable the plugin when not high enough.

I have this built into Ahoy, but don't forsee needing it in Popup Maker due to the above.

Let me know if there is anything I missed.

**Comment by rgadon107** [comment]
@danieliser Thanks for clarifying this for me. It teaches me a little more about how you build PUM. It occurred to me a few hours after I submitted the issue that since PUM is available in the WP.org PI repo, everything about the way that it's structured meets or exceeds the WP PI standards. 

I built a couple of plugins for my person use this past year, and included the WP version and PHP requirements in PI file header. I realize now that WP.org wouldn't require that as it's addressed in the README.txt file.
